src/integrations.py

```diff
@@ -124,9 +125,12 @@ def load(cls, requirer: TracingEndpointRequirer) -> "TracingData":
         if not (is_ready := requirer.is_ready()):
             return cls()
 
+        http_endpoint = urlparse(requirer.get_endpoint("otlp_http"))
+        scheme = "%s://" % http_endpoint.scheme
+
         return cls(
             is_ready=is_ready,
-            http_endpoint=requirer.otlp_http_endpoint(),  # type: ignore[arg-type]
+            http_endpoint=http_endpoint.geturl().replace(scheme, "", 1),  # type: ignore[arg-type]
```

Comment:
Just a minor thing of using the `f-string`. The old style string formatting `%` is error-prone.

`http_endpoint=http_endpoint.geturl().replace(f"{http_endpoint.scheme}://", "", 1)`